apps/src/dropletUtils.js

```diff
@@ -404,7 +417,15 @@ exports.generateAceApiCompleter = function (functionFilter, dropletConfig) {
         callback(null, []);
         return;
       }
-      callback(null, apis);
+      var line = session.getLine(pos.row);
+      var identifier = acUtil.retrievePrecedingIdentifier(line, pos.column);
+      if (pos.column > (identifier.length + 1) &&
+          line[pos.column - identifier.length - 1] === '.') {
```

Comment:
Ha!  I didn't know about the `2.toString()` edge case and workarounds.  That's pretty gross.